x/storage/types/crosschain.go

```diff
@@ -354,6 +440,18 @@ type CreateBucketSynPackage struct {
 }
 
 type CreateBucketSynPackageStruct struct {
+	Creator                        common.Address
+	BucketName                     string
+	Visibility                     uint32
+	PaymentAddress                 common.Address
+	PrimarySpAddress               common.Address
+	PrimarySpApprovalExpiredHeight uint64
+	PrimarySpApprovalSignature     []byte
+	ChargedReadQuota               uint64
+	ExtraData                      []byte
+}
+
+type CreateBucketSynPackageV2Struct struct {
```

Comment:
What's the difference between `CreateBucketSynPackageV2Struct` and `CreateBucketSynPackageV2`? Why we need both of them?

Reply:
`CreateBucketSynPackageV2Struct` has an extra field `GlobalVirtualGroupFamilyId`

Reply:
I mean `V2Struct` and `V2`.